### PR TITLE
Add the missing string to the frontend lib ini file

### PR DESCRIPTION
**Pull request by Bakual**
Removing unneeded space in a string introduced with https://github.com/joomla/joomla-cms/pull/13885 and copying it to frontend file (the lib ones have to be identic).

### Summary of Changes
See above

### Testing Instructions
Review

#### Disclaimer
Work is done by @brianteeman

### CODE REVIEW

File: administrator/language/en-GB/en-GB.lib_joomla.ini [hunk 1]
```diff
@@ -317,7 +317,7 @@ JLIB_FORM_SELECT_USER="Select a User"
 JLIB_FORM_VALIDATE_FIELD_INVALID="Invalid field: %s"
 JLIB_FORM_VALIDATE_FIELD_REQUIRED="Field required: %s"
 JLIB_FORM_VALIDATE_FIELD_RULE_MISSING="Validation Rule missing: %s"
-JLIB_FORM_VALIDATE_FIELD_URL_SCHEMA_MISSING="Invalid URL : URL schema is missing in %1$s. Please add one of the following at the beginning : %2$s."
+JLIB_FORM_VALIDATE_FIELD_URL_SCHEMA_MISSING="Invalid URL: URL schema is missing in %1$s. Please add one of the following at the beginning: %2$s."
```
Comment (chrisdavenport) [inline administrator/language/en-GB/en-GB.lib_joomla.ini]: "schema" should be "scheme".  See, for example, http://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml

File: language/en-GB/en-GB.lib_joomla.ini [hunk 2]
```diff
@@ -318,6 +318,7 @@ JLIB_FORM_VALIDATE_FIELD_INVALID="Invalid field: %s"
 JLIB_FORM_VALIDATE_FIELD_REQUIRED="Field required: %s"
 JLIB_FORM_VALIDATE_FIELD_RULE_MISSING="Validation Rule missing: %s"
 JLIB_FORM_VALUE_CACHE_APC="Alternative PHP Cache"
+JLIB_FORM_VALIDATE_FIELD_URL_SCHEMA_MISSING="Invalid URL: URL schema is missing in %1$s. Please add one of the following at the beginning: %2$s."
```
Comment (chrisdavenport) [inline language/en-GB/en-GB.lib_joomla.ini]: Ditto.

File: administrator/language/en-GB/en-GB.lib_joomla.ini (hunk 1 above)
Comment (Bakual) [inline administrator/language/en-GB/en-GB.lib_joomla.ini]: According to Brian schema is correct as it is the singular one (see https://twitter.com/brianteeman/status/837061254946697216)

File: language/en-GB/en-GB.lib_joomla.ini (hunk 2 above)
Comment (Bakual) [inline language/en-GB/en-GB.lib_joomla.ini]: Ditto 😄

**Comment by wilsonge** [comment]
@chrisdavenport let's get a separate PR done for the scheme to schema stuff

**Pull request merged**